build.gradle.kts

```diff
@@ -121,6 +123,27 @@ repositories {
 }
 
 tasks.withType<Wrapper> {
-    gradleVersion = "5.2.1"
+    gradleVersion = "5.4.1"
     distributionType = Wrapper.DistributionType.ALL
 }
+
+// https://github.com/ben-manes/gradle-versions-plugin#revisions
+tasks.named<DependencyUpdatesTask>("dependencyUpdates") {
+    resolutionStrategy {
+        componentSelection {
+            all {
+                val rejected = listOf("alpha", "beta", "rc", "cr", "m", "preview", "b", "ea").any { qualifier ->
+                    candidate.version.matches(Regex("(?i).*[.-]$qualifier[.\\d-+]*"))
+                }
+                if (rejected) {
+                    reject("Release candidate")
```

Comment:
Is it supposed to always reject here with `Release candidate`?

Reply:
It just changes what the report displays, not what the thing actually does. Having all of the rejections grouped together ("pre-release" maybe?) makes sense.

It's just copied directly from the docs (as commented).